wordpress-mobile/WordPress-Android
# Refactor Epilogue Screen

Updated Epilogue screen with a builder , latest copy and formatting

Fixes #14899 and #14912 

<table><tr>
<td><img width=360 src="https://user-images.githubusercontent.com/990349/123372328-b614f600-d5c6-11eb-9c2d-91230b2815cb.png" /></td>
<td><img width=360 src="https://user-images.githubusercontent.com/990349/123234367-3b909b80-d51e-11eb-84ec-5e4dd9a641f3.png" /></td>
<td><img width=360 src="https://user-images.githubusercontent.com/990349/123234383-3df2f580-d51e-11eb-8276-c8e8ce8ff152.png" /></td>
</tr></table>

To test:


- Turn on the blogging reminders flag (Me/App settings/Test feature configuration - BloggingRemindersFeatureConfig)
- Notice the blogging goals option appears (My Site > Settings > Blogging goals)
- Click on for e.g. "Twice a week" button under Blogging goals setting
- Notice the blogging reminders prologue screen is shown
- Click on "Set goals"
- Select days screen is shown, select days, then "Notify me" button is enabled
- Click on "Notify me" button
- Notice the epilogue screen is shown. Done button is enabled, Clicking Done button dismisses bottomsheet
- Go back to Settings and de-select all days and see that epilogue shows a "no reminders set" message
- Go back to Settings and select all days and see that epilogue show "everyday" message


## Regression Notes
1. Potential unintended areas of impact
Site Settings

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Blogging Reminders flow

3. What automated tests I added (or what prevented me from doing so)
Unit tests

PR submission checklist:
